src/test/kotlin/org/rust/ide/annotator/RsErrorAnnotatorTest.kt

```diff
@@ -2099,7 +2099,7 @@ class RsErrorAnnotatorTest : RsAnnotatorTestBase(RsErrorAnnotator::class) {
     """)
 
     @MockRustcVersion("1.38.0-nightly")
-    fun `test leading | in or patterns 1`() = checkErrors("""
+    fun `test leading vertical bar in or patterns 1`() = checkErrors("""
```

Comment:
Oh, it's interesting fact about name of `|`. I haven't known about it before.
Probably `pipe` is more known name for `|`